corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -466,13 +506,21 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
                 </div>
             </div>
             <div class="modal-footer">
-                <button type="button"
-                        class="btn btn-default"
-                        data-bind="click: $root.unsetRoleBeingEdited">
-                    {% trans "Cancel" %}
-                </button>
-                <div data-bind="saveButton2: $root.modalSaveButton.state,
-                                saveOptions: $root.modalSaveButton.saveOptions"></div>
+                {% if can_edit_roles %}
+                    <button type="button"
+                            class="btn btn-default"
+                            data-bind="click: $root.unsetRoleBeingEdited">
+                        {% trans "Cancel" %}
+                    </button>
+                    <div data-bind="saveButton2: $root.modalSaveButton.state,
```

Comment:
Realize you didn't add this, but the idea that we have a `saveButton` and a `saveButton2` is frightening, do you happen to know the differences?

Reply:
no i didn't really dig into the references. I mainly just tried to keep mostly everything as is and change what needed to get changed to constrain the scope of the work

Reply:
Yeah, definitely out of scope for this PR.